package.json

```diff
@@ -28,7 +28,7 @@
   "author": "Brigade Engineering",
   "license": "MIT",
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0"
```

Comment:
I think this might be considered a breaking change, which would require a semver major bump.